tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -13,20 +13,29 @@ namespace D2L.CodeStyle.Analyzers.Tests.Async.Generator;
 internal sealed class AsyncToSyncMethodTransformerTests {
 	[Test]
 	public void BasicTask() {
-		var actual = Transform( @"[GenerateSync] async Task HelloAsync( int abc, string def ) { await Task.Delay( 3 ); }" );
+		var actual = Transform( @"[GenerateSync] async Task HelloAsync( int abc, string def ) { return; }" );

```

Comment:
I would like the tests to have decent looking examples, but now that we aren't transforming things into `throw null` we can only use what we support.